src/Components/v2/CarouselV3.tsx

```diff
@@ -24,7 +34,7 @@ interface CarouselProps {
   /**
    * The render callback returns an image to display
    */
-  render: (slide) => React.ReactNode
+  render?: (slide, index) => React.ReactNode
```

Comment:
I believe this `render` is required

Reply:
And only takes a `slide` param looks like